### PR TITLE
LRQA-69761

### PR DESCRIPTION
**Pull request by leticiachiu**
Related Issue: https://issues.liferay.com/browse/LRQA-69761

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -2146,8 +2146,9 @@ definition {
 
 	@priority = "4"
 	test SubmitFormWithMultipleFieldsAndViewEntriesViaDescriptiveDisplay {
-		property portal.acceptance = "true";
-		property portal.upstream = "true";
+		property portal.acceptance = "false";
+		property portal.upstream = "false";
+		property portal.upstream = "quarantine";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:2151]: @leticiachiu, do we need `false` and `quarantine` for upstream?

**Pull request closed without merging**